skycultures/ticuna/constellation_names.eng.fab

```diff
@@ -0,0 +1,4 @@
+001	"Baweta" _("Turtle Collective")
+002 "Wücütcha" _("Wücütcha’s")
```

Comment:
Is "Wücütcha" term do not have English analog?

Reply:
The "Wücütcha" is a proper name of a myth of the Ticuna Indians